modules/google-maps/src/utils.ts

```diff
@@ -99,17 +108,23 @@ export function getViewPropsFromOverlay(map, overlay) {
   const projection = overlay.getProjection();
 
   const bounds = map.getBounds();
+  if (!bounds) {
+    return {width, height, left: 0, top: 0};
+  }
+
   const ne = bounds.getNorthEast();
   const sw = bounds.getSouthWest();
   const topRight = projection.fromLatLngToDivPixel(ne);
   const bottomLeft = projection.fromLatLngToDivPixel(sw);
+  assert(topRight && bottomLeft);
```

Comment:
In this code base we only use `assert`s to notify the user that there is something wrong with their usage pattern. In this context there is no clear action item for the user to fix the error if and when it occurs. `assert` is not a convenience to work around TypeScript errors. Consider casting the output of `fromLatLngToDivPixel` to a non-null type unless there is a legitimate reason that this might break in runtime.

Reply:
Note taken. Changing it in all pending PRs of mine.

Reply:
In this case, i'll follow @felixpalmer [suggestion](https://github.com/visgl/deck.gl/pull/6835#discussion_r853088637) and return _minimal_ props that should work.